schematics/component/index.ts

```diff
@@ -47,8 +119,7 @@ function addImportToNgModule(options: ComponentOptions): Rule {
       }
     }
     host.commitUpdate(importRecorder);
-    return host;
```

Comment:
I presume it wasn't necessary to return host... my bad 😛

Reply:
Host should not to return here. We return the host though further up on line 36, after the modifying the fs.